### PR TITLE
Feat/921 change icons doc

### PR DESCRIPTION
**Pull request by veilvokay**

### CODE REVIEW

**Review by thrbnhrtmnn — COMMENTED** [review]
We should elaborate a bit more how changing default icons actually can be done.

As I understand it, there are multiple ways like for example replacing a default icon but keeping the name or by defining a new default icon within the component.

Please also get 2 other approvals, as I can not verify if the process you described to update the icon is sufficient or not.

File: README.md [hunk 1]
```diff
@@ -121,6 +125,57 @@ Note: This is not filled out yet! ...
 ### Usage
 Note: This is not filled out yet! ...
 
+## :framed_picture: Icons Set
+
```
Comment (angsherpa456) [inline README.md:129]: Please maintain consistency without the new lines after the headings

File: README.md [hunk 2]
```diff
@@ -21,7 +21,11 @@
     - [Installation](#installation)
     - [Setup](#setup)
     - [Usage](#usage)
-  - [:white_check_mark: How to test](#white_check_mark-how-to-test)
+  - [:framed_picture: Icons Set](#framed_picture-icons-set)
+    - [Structure](#structure)
+    - [Adding Icons](#adding-icons)
+    - [Removing Icons](#removing-icons)
+  - [:white\_check\_mark: How to test](#white_check_mark-how-to-test)
```
Comment (angsherpa456) [inline README.md:28]: "\" is not required, can be just :white_check_mark:

File: README.md (hunk 1 above)
Comment (veilvokay) [reply README.md:129]: @angsherpa456 there is always a new line after a '##' heading

**Review by ChristianHoffmannS2 — APPROVED** [review]

**Comment by thrbnhrtmnn** [comment]
Hey @veilvokay , I am ready to approve the task, but we should merge the PR #952  from @davidken91 first. This may lead to some conflicts in your branch, as David also worked on the readme. This is why I will wait with the approval.

**Review by davidken91 — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -122,6 +127,117 @@ In addition, you might need to install Corepack. For more information checkout [
...
+Follow these steps to add your first icon:
+
+> Note: B01LER only supports icons in black and white.
+
+1. Navigate to `/icon-set` folder, paste your icon here (with all according resolutions) and run:
+
+   ```sh
+   $ yarn compile:icons
+   ```
+2. To check that your icon has been added run your porject locally and inspect the Icons tab:
```
Comment (davidken91) [inline README.md:154]: Small typo here

File: README.md [hunk 4]
```diff
@@ -122,6 +127,117 @@ In addition, you might need to install Corepack. For more information checkout [
...
+
+To check that changes have applied do the following:
+
+1. Run you project locally with:
+
+   ```sh
+   $ yarn start
+   ```
+
+2. Navigate to the Checkbox tab in your loval Storybook
```
Comment (davidken91) [inline README.md:214]: Small typo here too

File: README.md [hunk 5]
```diff
@@ -122,6 +127,117 @@ In addition, you might need to install Corepack. For more information checkout [
...
+- Number Input
+- Select
+- TabBar
+- Text Button
+- Text Input
+- Toggle Switch
+
+> Note: If you use any of those components, make sure that you either haven't removed the icons they use from the project, or replaced missing icons with yours.
+
+To remove any icon, sinmply navigate to `/icon-set` folder, delete icons you don't need in all resolutions and run:
```
Comment (davidken91) [inline README.md:235]: Small typo. Looks good to me other than the small spelling mistakes

**Comment by thrbnhrtmnn** [comment]
Hey @veilvokay , #952 has just been merged and leads to some conflicts. Once these are resolved  and we have 2 approvals again, we can also merge your branch :-)

**Review by JpunktWpunkt — APPROVED** [review]

**Review by davidken91 — APPROVED** [review]

**Pull request merged**